en/docs/deploy/monitor/system-statistics.md

```diff
@@ -78,14 +77,14 @@ Use the **Statistics Configuration** panel to configure the statistics
 view.
 
 1.  Enter values into the appropriate fields:
-    -   **Statistics Refresh Interval (ms)** - Allows you to specify the
-        statistics refresh rate.
-    -   **Response Time Graph** - Allows you to specify the X and Y
-        parameters of the Response Time graph.
+    -   **Statistics Refresh Interval (ms)** - Allows to specify the
```

Comment:
```suggestion
    -   **Statistics Refresh Interval (ms)**: Allows to specify the
```

Reply:
Refer https://github.com/wso2/docs-is/pull/2645#discussion_r645517587